infra/frontend/app-config/env-config/outputs.tf

```
@@ -26,3 +26,15 @@ output "incident_management_service_integration" {
 output "domain" {
   value = var.domain
 }
+
+output "sendy_api_key" {
+  value = var.sendy_api_key
```

⭐⭐  &nbsp; _(optional but seriously consider)_

```suggestion
  value = var.sendy_api_key
  sensitive = true
```

---

just a heads up that I had to make the upstream environment_configs output sensitive as well.